src/components/Comments/CommentEdit/CommentEdit.jsx

```diff
@@ -1,14 +1,55 @@
-import React from "react";
-import PropTypes from "prop-types";
-import { MDBIcon } from "mdbreact";
-import "./CommentEdit.scss";
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MDBIcon } from 'mdbreact';
+import './CommentEdit.scss';
 
-const CommentEdit = ({ clickReply, clickDelete, clickEdit, id }) => {
+const CommentEdit = ({
+  likes,
+  clickLike,
+  clickDisLike,
+  clickReply,
+  clickDelete,
+  clickEdit,
+  id
+}) => {
   const handleKeyUp = () => {};
-  
+  let disl = 'text-primary m-1';
```

Comment:
preferably rename this variable `dislikeClass`

Reply:
ok let me fix it